pkg/config/generate.go

```diff
@@ -376,37 +376,37 @@ func Generate(cs *acsapi.ContainerService, c *Config) (err error) {
 		kubeconfig **v1.Config
 	}{
 		{
-			clientKey:  c.OpenShiftMasterKey,
-			clientCert: c.OpenShiftMasterCert,
+			clientKey:  c.Certificates.OpenShiftMaster.Key,
+			clientCert: c.Certificates.OpenShiftMaster.Cert,
 			endpoint:   internalAPIServerHostname(cs),
 			username:   "system:openshift-master",
 			kubeconfig: &c.MasterKubeconfig,
 		},
 		{
-			clientKey:  c.BootstrapAutoapproverKey,
-			clientCert: c.BootstrapAutoapproverCert,
+			clientKey:  c.Certificates.BootstrapAutoapprover.Key,
```

Comment:
We can get rid of this now that we are back to a VM deployment and use the service account directly. Please open a follow-up issue.

Reply:
https://github.com/openshift/openshift-azure/issues/158